pkgs/applications/misc/p2pool/default.nix

```diff
@@ -19,20 +19,22 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "p2pool";
-  version = "2.7";
+  version = "3.1";
 
   src = fetchFromGitHub {
     owner = "SChernykh";
     repo = "p2pool";
     rev = "v${version}";
-    sha256 = "sha256-j3SVwat/LGw/iGcyNn8acR29Ob/WXDKyeCfDTsH+gxA=";
+    sha256 = "sha256-yHxg/9QhaDNlUFzylftsJEk+pJoSoTfA0rJfcolBdTs=";
     fetchSubmodules = true;
   };
 
   nativeBuildInputs = [ cmake pkg-config ];
   buildInputs = [ libuv zeromq libsodium gss hwloc openssl curl ]
     ++ lib.optionals stdenv.isDarwin [ Foundation ];
 
+  cmakeFlags = ["-DWITH_LTO=OFF"];
```

Comment:
Shouldn't this be removed?

Reply:
I don't reckon so. The comment above was referring to a `.patch` file which was removed in the force-push from https://github.com/NixOS/nixpkgs/commit/7e3decb417147f4a1d995bd15e375dd04ca0ca3c to https://github.com/NixOS/nixpkgs/commit/d86c897423d8a5416d5a02b66468407edf36cf9e.